src/twister2/report/helper.py

```diff
@@ -33,6 +33,13 @@ def get_item_type(item: pytest.Item) -> str:
     return ''
 
 
+def get_item_skip(item: pytest.Item) -> str:
+    """Return skip reason if test should be skipped."""
+    if marker := item.get_closest_marker('skip'):
```

Comment:
It may happened there will be more than one `pytest.mark.skip` added to a tests, and this function will return only first occurrence. Same problem will be in `get_item_platform` (no problem with yaml tests, but for python tests user can add more platforms)

Reply:
yes, I agree, if `skip` is used in testcase.yml we can lose info about quarantine, but skip in testcase is obsolete, I found it only in 5 yamls. However, I will try to change it with marking quarantined scenarios in separate plugin.